sdks/python/tox.ini

```diff
@@ -287,3 +287,12 @@ deps =
 commands =
   yapf --version
   time yapf --diff --parallel --recursive apache_beam
+
+[testenv:py3-dependency-check]
+# TODO: botocore, a part of [interactive], wants docutils<0.16, but Sphinx
+# pulls in the latest docutils. Uncomment this line once botocore does not
+# conflict with Sphinx:
+# extras = docs,test,gcp,aws,interactive,interactive_test
+extras = test,gcp,aws,interactive,interactive_test
+commands =
+  time {toxinidir}/scripts/run_dependency_check.sh
```

Comment:
I think you need to add `passenv = WORKSPACE` here, otherwise the script fails because WORKSPACE isn't set.